src/components/SEO/index.tsx

```diff
@@ -25,6 +25,7 @@ const SEO: React.FC<ISEOProps> = ({
   title,
   defaultMetaTitle,
   description,
+  site_name,
```

Comment:
site_name -> siteName (please, check the style around)

Reply:
where does site name come from?

Reply:
siteName is one of the meta property as like siteTitle, so i have declared it on ISEO props and thereafter defined it below.